src/styles/about.scss

```diff
@@ -1,47 +1,89 @@
-img {
-    margin-top: 0.5rem;
-    width: auto;
-    width: 40vh;
-    margin-right: 3rem;
-}
-
 .about-the-author {
+  margin-bottom: 1rem;
+
+  @media (min-width: 700px) {
     display: flex;
     justify-content: space-between;
-    padding: 2rem 30rem 10rem 30rem;
-    h1 {
-        padding-bottom: 1rem;
+    padding: 2rem 5%;
+    align-items: flex-start;
+  }
+
+  @media (min-width: 1200px) {
+    align-items: center;
+  }
+
+  h1 {
+    margin-top: 1rem;
+    padding-bottom: 1rem;
+    @media (min-width: 700px) {
+      margin-top: 0;
+    }
+  }
+
+  .author-image {
+    @media (min-width: 700px) {
+      margin-right: 3rem;
+      flex: 1 0 40%;
+      margin-left: 10%;
+      max-width: 360px;
```

Comment:
`flex: 1 0 40%` means:
- flex-grow: 1 // yes, let me expand to fill space
- flex-shrink: 0 // no, do not let me shrink below my basis
- flex-basis: 40% // make me 40% of this container. In this case, make me at least 40%, and larger if possible.

Flex basis is powerful and can be confusing. It's like using width, but can get you a better result in a flex container.

By setting `margin-left: 10%`, plus the 40% of our flex-basis, this container is 50% of the parent.